CONTRIBUTING.md

```diff
@@ -1,70 +1,105 @@
-# Contribution Guidelines
+Contributing to Shields
+=======================
 
-This is the home of Shields.io, home to the badge design specification, API documentation, and server code for Badges as a Service.
+Shields is a community project! We invite your participation through issues
+and pull requests.
 
-We invite participation through [GitHub Issues][], which we use much like a discussion forum. This repository should only contain non-implementation specific topics: specifications, design, and the web site.
 
-## This implementation
+Ways you can help
+-----------------
 
-Please see [INSTALL.md][] for information on how to start contributing code to
-shields.io.
-You can read a [Tutorial on how to add a badge](doc/TUTORIAL.md).
+### Contributing code
 
-[INSTALL.md]: ./INSTALL.md
+This project has quite a backlog of suggestions! If you're new to the project,
+maybe you'd like to opan a pull request to address one of them:
 
-Note that the root gets redirected to <http://shields.io>.
-For testing purposes, you can go to `http://localhost/try.html`.
+[![GitHub issues by-label](https://img.shields.io/github/issues/badges/shields/beginner-friendly.svg)]()
 
-## Ground rules
+Or you can adopt one of these pull requests:
 
-- The left-hand side of a badge should not advertize. It should be a noun
-  describing succinctly the meaning of the right-hand-side data.
-- New query parameters (such as `?label=` or `?style=`) should apply to any
-  requested badge. They must be registered in the cache (see `LruCache` in
-  `server.js`).
-- The format of new badges should be of the form
-  `/VENDOR/SUBVENDOR-BADGE-SPECIFIC/PARAMETERS.format`. For instance,
-  `https://img.shields.io/gitter/room/nwjs/nw.js.svg`. The vendor is gitter, the
-  badge is for rooms, the parameter is nwjs/nw.js, and the format is svg.
+[![GitHub issues by-label](https://img.shields.io/github/issues-pr/badges/shields/beginner-friendly.svg)]()
```

Comment:
This could link to https://github.com/badges/shields/pulls?q=is%3Apr+is%3Aopen+label%3Abeginner-friendly. What do you think this about it?